### PR TITLE
styling changes

### PR DESCRIPTION
**Pull request by CheeseMistress**

### CODE REVIEW

**Comment by Danondso** [comment]
The mobile view is looking wonky. 
image

**Comment by CheeseMistress** [comment]
* modified to adjust styling with screen size changes
* Content now fits within box or has no box depending on screen size.
* viewed checked out for all devices available in Chrome mobile testing

File: src/routes/index.svelte [hunk 1]
```diff
@@ -11,10 +15,36 @@
...
+        position: absolute;
+        top: 20%;
+        word-wrap: break-word;
+    }
+
+    @media only screen 
+    	and (max-width: 715px) {
+	    	main { 
+          width: 90vw; 
+          }
```
Comment (Danondso) [inline src/routes/index.svelte]: is this formatted this way or is this github?

File: src/routes/index.svelte [hunk 2]
```diff
@@ -11,10 +15,36 @@
...
     }
+
+    @media only screen
+      and (min-width: 716px) {
+        main {
+        width: 50vw;
+        font-family: Fredoka;
```
Comment (Danondso) [inline src/routes/index.svelte]: Looks like you have duplicate styling in here.

**Review by Danondso — APPROVED** [review]

**Pull request merged**